### PR TITLE
Add further information on the project to README

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-1000)

### CODE REVIEW

**Comment by richtier** [comment]
@stevenburnell-dit removed because `npm run sass-watch` never worked and was removed from script file months ago - the documentation had not been updated though.

File: README.md [hunk 1]
```diff
@@ -1,26 +1,47 @@
...
+
+This is the repo for Find A Buyer (FAB) - the Department for International Trade (DIT) service for exposing motivated British exporters to international buyers seeking British goods and service.
 
 ## Build status
 
 [![CircleCI](https://circleci.com/gh/uktrade/directory-ui-buyer/tree/master.svg?style=svg)](https://circleci.com/gh/uktrade/directory-ui-buyer/tree/master)
 
+## Development 
+
+The back-end framework is Django 1.9. The front-end uses minimal Javascript. The motivation for this is for accessibility reasons, to reduce technical complexity, and reduce cross-browser compatibility issues. Therefore most front-end work will be HTML and SASS/CSS development.
```
Comment (monikasulik) [inline README.md]: I would mention something about GDS here as well (with maybe a link to the GDS principles).

File: README.md [hunk 2]
```diff
@@ -46,42 +67,35 @@ Requires all host environment variables to be set.
...
+    $ virtualenv .venv -p python3.5
+    $ source .venv/bin/activate
+    $ pip install -r requirements_text.txt
 
+### Running the webserver
+	$ source .venv/bin/activate
     $ make debug_webserver
 
-### Run debug tests
+### Runing the tests
```
Comment (monikasulik) [inline README.md]: typo: Running not Runing

File: README.md [hunk 3]
```diff
@@ -46,42 +67,35 @@ Requires all host environment variables to be set.
...
 
-    $ make debug_test
+    $ make debug_pytest
```
Comment (monikasulik) [inline README.md]: I think `debug_test` is the better way to run tests actually. You can mention `debug_pytest` as a way to skip the flake8 stuff which might be useful on slower machines (for me flake8 only means a few seconds more, but if it's running slowly for somebody then it's probably easier to do just debug_pytest most of the time).

**Pull request merged**